core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2374,6 +2380,28 @@ public void setScmCheckoutRetryCount(int scmCheckoutRetryCount) throws IOExcepti
         save();
     }
 
+    /**
+     * Gets the check interval for computer retention.
+     *
+     * @since TODO
+     */
+    public int getComputerRetentionCheckInterval() {
+        return computerRetentionCheckInterval;
+    }
+
+    /**
+     * Updates the check interval for computer retention and restarts the check cycle.
+     *
+     * @param interval new check interval in seconds
+     * @throws IOException saving config to disk failed
+     * @since TODO
+     */
+    public void setComputerRetentionCheckInterval(int interval) throws IOException {
+        this.computerRetentionCheckInterval = interval;
```

Comment:
Does not prevent submissions with the value 0.

Reply:
Fixed as part of moving the variable into the model dab113783ecfdf84685bcf8ead4c178c5fdc6c5f.